source/index.rst

```diff
@@ -1,4 +1,21 @@
 the iceoryx2 book
 =================
 
-Coming soon !
+.. toctree::
+   :maxdepth: 1
+   :caption: User Manual
+
+   quickstart
+   introduction
+   fundamentals/index
+   tutorials/index
+   glossary
+
+.. toctree::
+   :maxdepth: 1
+   :caption: API Documentation
+
+   Rust <https://docs.rs/iceoryx2/latest>
+   C <https://eclipse-iceoryx.github.io/iceoryx2/c/latest>
+   C++ <https://eclipse-iceoryx.github.io/iceoryx2/cxx/latest>
+   Python <https://eclipse-iceoryx.github.io/iceoryx2/python/latest>
```

Comment:
These links are currently not working because the "latest" URL is not provided by the iceoryx page. Where is https://eclipse-iceoryx.github.io/iceoryx2/ hosted so we can fix it there hopefully.